packages/framework/fluid-static/src/utils.ts

```diff
@@ -3,35 +3,36 @@
  * Licensed under the MIT License.
  */
 
+import type { DataObjectKind } from "@fluidframework/aqueduct/internal";
```

Comment:
I don't really know one way or the other, but if asked I would have thought that fluid-static was a lower layer than aqueduct and shouldn't import from it. I would consult @ChumpChief to see if he has thoughts about it.

Reply:
You might think that, but our declarative API surface is actually build on top of aqueduct, and lives in the fluid-static package. This might not be the best package naming but that's how its layered, and we already depend on aqueduct here because of it, just not directly in this specific file until now. See packages/framework/fluid-static/src/rootDataObject.ts which depends on several aqueduct imports.

Since DataObjectKind in internal, we could move it later if we ever wany to remove this dependency and adjust layering.